pkg/volume/azure_dd/azure_dd.go

```diff
@@ -145,7 +145,7 @@ func (plugin *azureDataDiskPlugin) GetVolumeLimits() (map[string]int64, error) {
 		// default values from here will mean, no one can
 		// override them.
 		glog.Errorf("failed to get azure cloud in GetVolumeLimits, plugin.host: %s", plugin.host.GetHostName())
-		return volumeLimits, nil
+		return nil, fmt.Errorf("No cloudprovider present")
```

Comment:
`getCloud()` returns a more descriptive error; would it make sense to return `err` instead?

Also, I'm not sure if I understand the last sentence of the above comment, but shouldn't it be removed as well?

Reply:
about the comment - comment is in-fact still correct. We should not be returning default value from this function if we can't fetch Azure Cloudprovider.